src/main/java/com/split/impl/PercentSplitStrategy.java

```diff
@@ -0,0 +1,56 @@
+package com.split.impl;
+
+import com.models.Borrow;
+import com.models.Transaction;
+import com.split.SplitStrategy;
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Created by bugkiller on 21/09/19.
+ */
+public class PercentSplitStrategy implements SplitStrategy {
+
+  private static DecimalFormat decimalFormat = new DecimalFormat(".##");
+
+  public PercentSplitStrategy() {
+    decimalFormat.setRoundingMode(RoundingMode.UP);
+  }
+
+  @Override
+  public Transaction split(String inputLine) {
+    String split[] = inputLine.split("PERCENT");
```

Comment:
Should have parsed input format outside the core design. You could have parsed and created an expense object and passed it inside the split method.